projects/components/src/quick-search/quick-search.component.html

```diff
@@ -23,6 +23,9 @@
             >
                 <h5 *ngIf="showSectionTitle(searchSection)" class="search-result-section-title">
                     {{ searchSection.provider.sectionName }}
+                    <span *ngIf="partialResult(searchSection); let partial" class="partial-result">
+                        {{ 'vcd.cc.quickSearch.pagination' | translate: partial }}
```

Comment:
What about making `'vcd.cc.quickSearch.pagination'` to be an input property (message key) of the provider?

Reply:
I think we can add that later since we don't really need it now